### PR TITLE
Separates beer googles into science glasses and beer goggles. Sprites for each! 

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request

Beer goggles are now a unique item that only the bartender as they are now uncraftable and can not be removed. Science goggles take that role and function the exact same as beer goggles except they do not allow you to throw drinks without spilling and look rather tacky.
Glasses 1
image
image
image

A lot more work than the original PR but this is a lot better

## Why It's Good For The Game

It's quite lame that sunglasses and beer goggles are different items and yet they are indistinguishable from each other. And they look neat.   

## Changelog
:cl: itseasytosee
add: Crafting with sunglasses and science goggles now creates science glasses that are identical to beer goggles but don't allow the throwing of drinks without spilling and have a purple tint.
imageadd: Sprite for science glasses.
imageadd: New sprite for beer goggles.
:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
This is some ugly shit

**Comment by FantasticFwoosh** [comment]
Not a fan of the magneta, wouldnt call it ugly but it does look tacky.

**Comment by necromanceranne** [comment]
Looks fine to me, maybe with some purple on the lens itself and a lighter shade of purple for the reflection.

**Comment by Tlaltecuhtli** [comment]
why not brown

**Comment by HomocidlePancakes** [comment]
Yeah, brown would be a lot more better.

**Comment by itseasytosee** [comment]
Sounds good, I just kinda defaulted to purple because science goggles are purple.

**Comment by trollbreeder** [comment]
Looks a bit off with the red with magenta line design. This is moving in the right direction though. SCUNGLASSES!

**Comment by carlarctg** [comment]
looks tacky and not bartendy

**Comment by itseasytosee** [comment]
Sounds good, I kinda defaulted to purple because science goggles are purple.
I am tempted to add the purple variant to the bardrobe, but that adds a new pair of sunglasses into circulation.
As a final option, I could make the default glasses the barman starts with brown and the one crafted with science goggles purple.

**Comment by itseasytosee** [comment]
I also plan on taming that ugly magenta line.

**Comment by improvedname** [comment]
The purple once since science goggles are purple

**Comment by itseasytosee** [comment]
Yeah, that's currently what it is using. But I'm considering making two items beer goggles being unique to the barman and science glasses or any other creative name gets thrown at me be the craftable item.

**Comment by PKPenguin321** [comment]
Brown looks better and is beer colored so it kinda fits

**Comment by itseasytosee** [comment]
Brown it is

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -211,6 +212,12 @@
 	. = ..()
 	REMOVE_TRAIT(user, TRAIT_BOOZE_SLIDER, CLOTHING_TRAIT)
 
+/obj/item/clothing/glasses/sunglasses/chemical
+	name = "science glasses"
+	icon_state = "sunhudsci"
+	desc = "A pair of tacky purple sunglasses that allow the wearer to recognize various chemical compounds with only a glance."
```
Comment (itseasytosee) [inline code/modules/clothing/glasses/_glasses.dm]: Thanks guys for your ideas

Comment (cacogen) [reply code/modules/clothing/glasses/_glasses.dm]: should be science HUDSunglasses

Comment (cacogen) [reply code/modules/clothing/glasses/_glasses.dm]: should go in this file
https://github.com/tgstation/tgstation/blob/master/code/modules/clothing/glasses/hud.dm

**Comment by HomocidlePancakes** [comment]
Much better.

File: code/modules/clothing/glasses/_glasses.dm (hunk 1 above)
Comment (trollbreeder) [reply code/modules/clothing/glasses/_glasses.dm]: As long as you dont name them **scunglasses** i will not rest

Comment (itseasytosee) [reply code/modules/clothing/glasses/_glasses.dm]: #scun# as in meth?

**Review by kriskog — CHANGES_REQUESTED** [review]
Couple rogue pixels you gotta take care of
image

**Comment by itseasytosee** [comment]
thanks

**Review by kriskog — APPROVED** [review]

File: code/modules/clothing/glasses/_glasses.dm (hunk 1 above)
Comment (trollbreeder) [reply code/modules/clothing/glasses/_glasses.dm]: Science + sunglasses = scunglasses

Comment (itseasytosee) [reply code/modules/clothing/glasses/_glasses.dm]: maybe in another life or PR

**Pull request merged**